tgui/packages/tgui/interfaces/Manufacturer/index.tsx

```diff
@@ -83,26 +83,30 @@ export const Manufacturer = (_, context) => {
   }
 
   // Get a ManufacturableData from a QueueBlueprint using its type, category, and name.
-  let queueBlueprintRefs = data.queue.map((queued:QueueBlueprint) =>
+  let queueBlueprintRefs = data.queue && data.queue.length
+  && data.queue.map((queued:QueueBlueprint) =>
     blueprints_by_category[queued.category].find((key) => (key.name === queued.name))
   );
```

Comment:
```suggestion
  const queueBlueprintRefs = (data.queue ?? []).map((queued:QueueBlueprint) =>
    blueprints_by_category[queued.category].find((blueprint) => (blueprint.name === queued.name))
  );
```
If the `key` -> `blueprint` change doesn't make sense, please explain what that would represent.